apps/myaccount/src/components/multi-factor-authentication/multi-factor-authentication.tsx

```diff
@@ -16,6 +16,8 @@
  * under the License.
  */
 
+// useRequiredScopes hook cannot be used inside myaccount, hence disabling the rule here.
```

Comment:
If this needs to be used in My Account I think we should move it to the `@wso2is/core`

Reply:
The reason we cannot use `useRequiredScopes` hook in my account is that in order to use it, the app should be wrapped with AccessControlProvider but it is tightly coupled with feature gate configs which are currently meant for console app [1].

Yet the eslint rule for prohibiting hasRequiredScopes has been added to the project root, resulting in eslint warnings being thrown in myaccount app as well.

> If this needs to be used in My Account I think we should move it to the @wso2is/core. Let's track that as a improvement.

Just decoupling feature gate configs from AccessControlProvider would be enough. I will create a refactoring task to address this.

[1] https://github.com/wso2/identity-apps/blob/master/apps/console/src/app.tsx#L323

Reply:
This improvement will be tracked with https://github.com/wso2/product-is/issues/22370